#!/bin/sh /usr/share/dpatch/dpatch-run
## 40_testmail_fix.dpatch by Henrique de Moraes Holschuh <hmh@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix testEmail button call

@DPATCH@
diff -urNad --exclude=CVS --exclude=.svn ./hpssd.py ./hpssd.py
--- ./hpssd.py	2005-09-17 00:43:27.000000000 -0300
+++ ./hpssd.py	2005-09-17 13:20:55.434360268 -0300
@@ -385,6 +385,8 @@
         except KeyError:
             log.debug("New device: %s" % device_uri)
 
+            # Use NULL address for envelope sender
+            from_address = '<>'
             try:
                 devices[device_uri] = device.ServerDevice(device_uri, hpiod_sock,
                                                            QueryModel, QueryString)
